feature/google/src/test/java/BillingWrapperTest.kt

```diff
@@ -110,7 +110,7 @@ class BillingWrapperTest {
         val billingClientPurchaseHistoryListenerSlot = slot<PurchaseHistoryResponseListener>()
         every {
             mockClient.queryPurchaseHistoryAsync(
-                any(),
+                any<String>(),
```

Comment:
this became ambiguous, as the new version of this function now takes `QueryPurchaseHistoryParams`